src/thrift_0_13_0/Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+THRIFT_VERSION = 0.13.0
+THRIFT_VERSION_FULL = $(THRIFT_VERSION)-6
+
+MAIN_TARGET = libthrift-$(THRIFT_VERSION)_$(THRIFT_VERSION_FULL)_$(CONFIGURED_ARCH).deb
```

Comment:
>libthrift [](http://example.com/codeflow?start=14&length=9)

Is it true that you only need to build it for Debian versions before bulleyes?

Reply:
It is better to build from sources for bulleyes too